state/workingset.go

```diff
@@ -63,6 +64,18 @@ type (
 	}
 )
 
+// Context provides the runactions with auxiliary information.
+type Context struct {
```

Comment:
In the future, we could think of implement the std context

Reply:
ok